.gitignore

```diff
@@ -130,4 +130,8 @@ dmypy.json
 
 
 # idea file
-.idea
\ No newline at end of file
+.idea
+
+#ignore environment k liye
+env/
```

Comment:
Env already included in line 104 gitignore file.

Reply:
Ok boss ...